CHANGELOG.md

```diff
@@ -11,6 +11,14 @@ but this project DOES NOT adhere to [Semantic Versioning](http://semver.org/).
 * Remove `LeanCode.AsyncTasks` and Hangfire-based implementation
 * `FindAndEnsureExistsAsync` now throws `EntityDoesNotExistException`
 * Remove `AddAsync`, `UpdateAsync`, `DeleteAsync` and `DeleteRangeAsync` from `EFRepository` and `IRepository`
+* Remove `IRepository<T>`
+* Change `EFRepository.Update` so that it does not call `dbContext.Update` anymore
+* Rework `FixedTimeProvider`
+* Add query's and operation's responses in logs
+* Rework `JsonLaxDateOnlyConverter`
+* Add `JsonLaxTimeOnlyConverter` and `JsonLaxDateTimeOffsetConverter`
+* Extract `StyleCop` out of `CodeAnalysis`
+* Use file-scoped namespaces across all projects
```

Comment:
This one does not affect users of these packages so it should not end up in (this part of) the changelog.